### DISCUSSION TITLE
[水帖]p30pro和mate20pro哪个更好？

### DISCUSSION THREAD

**Discussion by tianxianuo** [share121/inter-knot#1929] [其他]
![IMG_20240813_195747](https://github.com/user-attachments/assets/3aea6560-dc52-490b-baa4-8c6738f37816)
![IMG_20240812_171755](https://github.com/user-attachments/assets/62fcdfa5-1a86-4e2a-b4ee-bac230916029)
逛闲鱼心血来潮，想开学搞台手机收藏兼备用，绳匠们觉得哪台更好？(ÒωÓױ)！！！

**Comment by tianxianuo** [comment]
(O∆O)

**Comment by ShenYouJiaA** [comment]
收藏建议mate20pro 它标志着华子巅峰的开始 但p30pro有个五倍镜头手感也更好一点

**Comment by keerdun** [comment]
谁把我眼镜摘了？

**Comment by cpu644** [comment]
楼上+1

**Comment by 1451374815** [comment]
图片好糊

**Comment by Yitaruose-BaiXu** [comment]
用oppoA5享爆炸人生

**Comment by jinyushengg** [comment]
如果单纯想收藏就mate20pro，如果想还想玩玩拍照就p30pro，毕竟当初“人在摩天轮做，旁边p30pro在看”

**Comment by Buer-Nahida** [comment]
我推荐OPPO A5

**Comment by zhuongxt** [comment]
建议使用三星Note7

**Reply by zhuongxt** [reply]
好用到爆炸

**Comment by shenguojia123** [comment]
为啥这也被ban

**Comment by tianxianuo** [comment]
![IMG_20240815_105437](https://github.com/user-attachments/assets/fdbdb3cd-845b-4930-9a6a-a454d3b12fb6)
![IMG_20240815_105459](https://github.com/user-attachments/assets/c8b9f426-a79b-453b-9acc-963cbe99a8f1)
我也没想到绳网会更这个功能(｡･ˇ_ˇ･｡:)，图片是我手动模糊的，上面是原图
